tiledb/sm/consolidator/array_meta_consolidator.cc

```diff
@@ -86,30 +86,13 @@ Status ArrayMetaConsolidator::consolidate(
           QueryType::WRITE, encryption_type, encryption_key, key_length),
       throw_if_not_ok(array_for_reads.close()));
 
-  // Swap the in-memory metadata between the two arrays.
+  // "Swap" the in-memory metadata between the two arrays.
```

Comment:
We're not swapping any more so I'd reword this comment rather than use the implied finger quotes.